test/replutil.jl

```diff
@@ -227,8 +227,9 @@ let err_str,
     i = reinterpret(EightBitType, 0x54),
     j = reinterpret(EightBitTypeT{Int32}, 0x54)
 
-    err_str = @except_str Symbol() MethodError
-    @test contains(err_str, "MethodError: no method matching Symbol()")
+    #PR 16130 (Symbol)
+    #err_str = @except_str Symbol() MethodError
+    #@test contains(err_str, "MethodError: no method matching Symbol()")
```

Comment:
@tkelman I did not realize that here Symbol() is not important as Symbol per se. I will see how this works substituting Float64(), which generates a MethodError